mentoring-app/src/app/store/actions/course.actions.ts

```diff
@@ -0,0 +1,75 @@
+import { Action } from '@ngrx/store';
+import { CourseClass } from '../../courses-page/course/models/course.models';
+
+export enum CoursesActionTypes {
+    LoadCourses = '[Courses Page] Load Courses',
+    LoadCoursesSucess = '[Courses Page] Load Courses Sucess',
+    LoadMoreCourses = '[Courses Page] Load More Courses',
+    LoadMoreCoursesSucess = '[Courses Page] Load More Courses Sucess',
+    DeleteCourse = '[Courses Page] Delete Course',
+    GetCourse = '[Edit Page] Get Course',
+    GetCourseSucess = '[Edit Page] Get Course Sucess',
+    UpdateCourse = '[Edit Page] Update Course',
+    AddCourse = '[Add Page] Add Course'
+  }
+
+export class LoadCourses implements Action {
+  readonly type = CoursesActionTypes.LoadCourses;
+}
+
+export class LoadCoursesSucess implements Action {
```

Comment:
If there are LoadCourseSuccess, GetCourseSuccess, there should probably be LoadCourseFail and GetCourseFail?